pom.xml

```diff
@@ -2477,14 +2494,15 @@
                     As a result, this profile will not work in IntelliJ,
                     but we don't need JaCoCo in IntelliJ anyway.
                  -->
-                <surefire.jacoco.args>@{surefire.jacoco.args.lateEval}</surefire.jacoco.args>
-                <failsafe.jacoco.args>@{failsafe.jacoco.args.lateEval}</failsafe.jacoco.args>
+                <surefire.jvm.args.jacoco>@{surefire.jvm.args.jacoco.lateEval}</surefire.jvm.args.jacoco>
+                <failsafe.jvm.args.jacoco>@{failsafe.jvm.args.jacoco.lateEval}</failsafe.jvm.args.jacoco>
                 <!--
                     We don't use @{...} for late property evaluation here,
                     because these properties are evaluated by the maven-resource-plugin,
                     which doesn't understand the @{...} syntax but somehow manages to always use late evaluation.
                  -->
-                <test.container.jacoco.args>${failsafe.jacoco.args.lateEval}</test.container.jacoco.args>
+                <arquillian.wildfly.jvm.args.jacoco>${failsafe.jvm.args.jacoco.lateEval}</arquillian.wildfly.jvm.args.jacoco>
+                <pax-exam.jvm.args.jacoco>${failsafe.jvm.args.jacoco.lateEval}</pax-exam.jvm.args.jacoco>
```

Comment:
I think I would have kept only one jacoco variable but it's really a matter of taste.

Looks good otherwise.